code/modules/power/lighting.dm

```diff
@@ -749,3 +749,8 @@
 	on = FALSE
 	visible_message("<span class='danger'>[src] flickers and falls dark.</span>")
 	update(0)
+
+// TODO: Add creation methods for these
+/obj/machinery/light/floor
```

Comment:
Curious what these are for. Also that TODO.

Reply:
They are for the in-floor lights
![image](https://user-images.githubusercontent.com/25063394/90522105-3aa31680-e163-11ea-8371-16c90e068cda.png)

I can do the todo in this PR but it makes it bigger than it already is, and the lights are already an admin only thing. *Insert shadowling balance concern here too*